docs/core/index.md

```diff
@@ -7,137 +7,41 @@ ms.date: 06/20/2016
 ---
```

Comment:
ms.date should be updated here
add a ms.custom metadata with the value `ms.custom: "updateeachrelease"` given that you're mentioning specific releases so we need to put this on the radar for every release